web/index.js

```diff
@@ -169,17 +166,17 @@ function isValidRole(roleName, roleLabel) {
  * @param {Event} event - The most recent event object
  */
 function handleEvents(event) {
-  if (event.detail.roomId === roomId) {
+  const detail = event.detail;
+  if (detail.roomId === roomId) {
     bdk.log.debug('Event received: ', event);
-    if ((event.detail.context) &&
-      (event.detail.context.type === 'User')) {
+    if (detail.context && detail.context.type === 'User') {
```

Comment:
could we use the conditional ! that conals been using here?

Reply:
I dont know which version of what must be changed, do you know?

Reply:
node 14.0.0
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining